modules/ixBidAdapter.js

```diff
@@ -39,7 +39,7 @@ const BANNER_TIME_TO_LIVE = 300;
 const VIDEO_TIME_TO_LIVE = 3600; // 1hr
 const NATIVE_TIME_TO_LIVE = 3600; // Since native can have video, use ttl same as video
 const NET_REVENUE = true;
-const MAX_REQUEST_SIZE = 8000;
+const MAX_REQUEST_SIZE = 32000; // IX POST limit
```

Comment:
So you keep the splitting logic in the adapter?

Reply:
who does splitting mean here? you mean different size limits co exist due to different version of PBJS adopted by pubs?

Reply:
Yes, I think it is a good point. I will talk with my team on removing the redundant logic but for now we could keep this as another follow-up change after POST is successfully in live.